mock-thunderhead/src/main/java/com/sequenceiq/thunderhead/grpc/service/auth/MockUserManagementService.java

```diff
@@ -316,6 +318,9 @@
 
     private Optional<SshPublicKey> sshPublicKey;
 
+    @Value("${auth.mock.azure.disk.SSEWithCMk.enable}")
```

Comment:
Like for the field name, I wonder if the `k` at the end of `SSEWithCMk` should be also uppercase.